### PR TITLE
Hindi Translation added

### PR DESCRIPTION
**Pull request by purva98**
Me and my team-mate @riddhi99 have done Hindi translation.
Please let us know if any changes are required :)
Cheers!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased]
```
Comment (hovancik) [inline CHANGELOG.md]: You should not create new `## [Unreleased]` title, but use already existing one

**Comment by hovancik** [comment]
There are some merge conflicts, do you know how to handle those or should I do it?

**Comment by purva98** [comment]
I have solved the merge conflict , please review:)

**Comment by hovancik** [comment]
@purva98 I still see conflicts: 


screenshot_2018-02-15_19-04-49

Do you know how to handle it?

**Pull request closed without merging**